guide/english/certifications/javascript-algorithms-and-data-structures/regular-expressions/check-for-mixed-grouping-of-characters

```diff
@@ -0,0 +1,20 @@
+---
+title: Check For Mixed Grouping of Characters
```

Comment:
The goal of the exercise is to introduce grouping, yet none of the hints are about that.